static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -38,6 +38,12 @@ const SUBSCRIPTION_BANNER_CLOSED_KEY = 'subscriptionBannerLastClosedAt';
 const COMPONENT_TYPE = 'ACQUISITIONS_SUBSCRIPTIONS_BANNER';
 const OPHAN_EVENT_ID = 'acquisitions-subscription-banner';
 
+const getVariant = () =>
+    // pass variant to ga
+    // pass variant to ophan
+    Math.round(Math.random());
+
+console.log(getVariant());
```

Comment:
Do you want this left in @jfsoul ?

Reply:
Nope, removing a few things as a tidy up now.